fpga.chart.py

```diff
@@ -63,11 +65,18 @@ def __init__(self, configuration=None, name=None):
         self.fpga_count = 1
         self.dsn = self.configuration.get('dsn')
         self.check_temp_power = self.configuration.get('check_temp_power')
+        self.temp = None
+        self.power = None
         self.metrics = [ 'bytes', 'jobs', 'max' ]
 
         if self.check_temp_power:
             self.metrics.extend([ 'temps', 'powers' ])
 
+        if self.update_every < 10:
```

Comment:
You can use `self.temp_power_update_interval = 10 if self.update_every < 10 else self.update_every`